app/models/chatroom.rb

```diff
@@ -1,3 +1,17 @@
+# == Schema Information
+#
+# Table name: chatrooms
+#
+#  id         :bigint           not null, primary key
+#  name       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  gig_id     :bigint
+#
+# Indexes
+#
+#  index_chatrooms_on_gig_id  (gig_id)
+#
```

Comment:
As mentioned on the comment for the annotate gem, this is the kind of comment that is added. These comments are added/updated whenever a migration is run.